build-system/pr-check/checks.js

```diff
@@ -112,6 +112,13 @@ async function prBuildWorkflow() {
     timedExecOrDie('amp server-tests');
   }
 
+  if (
+    buildTargetsInclude(Targets.AVA) ||
+    buildTargetsInclude(Targets.RUNTIME)
+  ) {
+    timedExecOrDie('amp make-extension-e2e');
```

Comment:
```suggestion
    timedExecOrDie('amp make-extension --test');
```

Reply:
1. I didn't want to make `--test` run multiple tests since that would make flags incompatible. Instead, I added `--test` and `--cleanup`, and we run the command twice.

2. I've made the changes requested to `checks.js`, **but is it the right place?** This task seems more involved than other checks.

Reply:
The only other potential spots are with integration tests, or in a separate job. However, I think both alternatives are worse than doing it in `checks.js` because the `Checks` job has no dependencies and begins at the very start of the CI build. So adding an extra minute to it does not affect the overall round-trip time for CI. Therefore, leaving this here SGTM.

Reply:
sg!